src/main/groovy/com/stormpath/tck/logout/LogoutIT.groovy

```diff
@@ -52,24 +48,27 @@ class LogoutIT extends AbstractIT {
         assertTrue(isCookieDeleted(cookies.get("refresh_token")))
     }
 
+    // TODO - this is stormpath specific and as such is commented out for now
+    // An agnostic approach here might be to login with the access token and confirm that it fails
+    // and try to use the refresh token and confirm that it fails
     private void assertTokenIsRevoked(String tokenJwt, String resourceType) throws Exception {
```

Comment:
Consider having this test throw an exception just so that we know it's currently failing/needs to be refactored.